.github/container/Dockerfile.t5x.arm64

```diff
@@ -166,7 +175,7 @@ COPY --from=array_record-builder /tmp/array_record/all_dist/array_record*linux_a
 RUN echo "array_record @ file://$(ls /opt/array_record*.whl)" >> /opt/pip-tools.d/requirements-t5x.in
 
 COPY --from=grain-builder /tmp/grain/all_dist/grain*.whl /opt/
-RUN echo "grain @ file://$(ls /opt/grain*.whl)" >> /opt/pip-tools.d/requirements-t5x.in
+RUN echo "grain-nightly @ file://$(ls /opt/grain*.whl)" >> /opt/pip-tools.d/requirements-t5x.in
```

Comment:
Huh, good catch. I wonder why this didnt break things before?

Reply:
So I think this is what happened:
1. airio used to depend on grain-nightly==0.0.3 which wasn't platform specific on pypi, so we installed "grain" while airio installed "grain-nightly", so no conflicts b/c pip installed both packages:
```
commit 065ae20c9bbae3a6664f02b9b1bc91a3624a7ba8
Author: Michelle Casbon <michellecasbon@google.com>
Date:   Tue Jan 30 15:06:55 2024 -0800

    Pin grain-nightly to 0.0.3.

    PiperOrigin-RevId: 602852846

diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -40,31 +40,31 @@
     long_description=_LONG_DESCRIPTION,
     long_description_content_type='text/markdown',
     author='Google Inc.',
     author_email='no-reply@google.com',
     url='http://github.com/google/airio',
     license='Apache 2.0',
     packages=setuptools.find_packages(),
     include_package_data=True,
     scripts=[],
     install_requires=[
         'absl-py',
         'clu @ git+https://github.com/google/CommonLoopUtils#egg=clu',
-        'grain-nightly',
+        'grain-nightly==0.0.3',
```
2. It was only on Feb 9th that they switched to the 0.0.6 pin that was x86_64 specific, that it broke b/c now it now couldn't find the wheel on pypi:
```
commit 69ceecf5ba5b178a5bc08be28a30f6a71092966f
Author: Michelle Casbon <michellecasbon@google.com>
Date:   Fri Feb 9 04:23:18 2024 -0800

    Bump grain version to 0.0.6.

    PiperOrigin-RevId: 605592388

diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -40,31 +40,31 @@
     long_description=_LONG_DESCRIPTION,
     long_description_content_type='text/markdown',
     author='Google Inc.',
     author_email='no-reply@google.com',
     url='http://github.com/google/airio',
     license='Apache 2.0',
     packages=setuptools.find_packages(),
     include_package_data=True,
     scripts=[],
     install_requires=[
         'absl-py',
         'clu @ git+https://github.com/google/CommonLoopUtils#egg=clu',
-        'grain-nightly==0.0.3',
+        'grain-nightly==0.0.6',
```